src/main/java/edu/wpi/grip/core/operations/opencv/PointGeneratorOperation.java

```diff
@@ -0,0 +1,45 @@
+package edu.wpi.grip.core.operations.opencv;
+
+
+import com.google.common.eventbus.EventBus;
+import edu.wpi.grip.core.InputSocket;
+import edu.wpi.grip.core.OutputSocket;
+import edu.wpi.grip.core.SocketHint;
+import org.bytedeco.javacpp.opencv_core.Point;
+
+public class PointGeneratorOperation implements CVOperation {
```

Comment:
I think `NewPointOperation` would be a better name for this class (and similarly for the Size one)